scripts/parse-coverage.sh

```
@@ -0,0 +1,89 @@
+echo "{"
+echo "  \"branch\": \"$BRANCH\","
+echo "  \"commit\": \"$COMMIT\","
+echo "  \"timestamp\": \"$TIMESTAMP\","
+echo "  \"mode\": \"$MODE\","
+echo "  \"total_line\": ${TOTAL_LINE:-0.0},"
+echo "  \"total_func\": ${TOTAL_FUNC:-0.0},"
+echo "  \"total_branch\": null,"
+echo "  \"packages\": ["
+
+first=true
+while IFS='|' read -r pkg cov total pct; do
```

Shellcheck (0.11.0)

[warning] cov appears unused. Verify use (or export if used externally).

(SC2034)